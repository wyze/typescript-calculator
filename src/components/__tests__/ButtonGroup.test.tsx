import { h } from 'preact'
import ButtonGroup from '../ButtonGroup'
import snapshot from './utils/snapshot'

describe('<ButtonGroup />', () => {
  it('renders', () => snapshot(<ButtonGroup />))
  it('renders with children', () => snapshot(<ButtonGroup><span /></ButtonGroup>))
})
