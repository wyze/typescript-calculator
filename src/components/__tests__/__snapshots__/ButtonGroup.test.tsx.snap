// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonGroup /> renders 1`] = `
<div
  className="css-1cqgl9p"
/>
`;

exports[`<ButtonGroup /> renders with children 1`] = `
<div
  className="css-1cqgl9p"
>
  <span />
</div>
`;
